e2e-tests/portal/cypress.config.js

```diff
@@ -12,6 +14,8 @@ module.exports = defineConfig({
   tfmApiPort: '5004',
   // TODO: Read value from environment variable
   apiKey: 'test',
+  dbName: 'dtfs-submissions',
+  dbConnectionString: 'mongodb://root:r00t@localhost:27017/?authMechanism=DEFAULT',
```

Comment:
I think the top level .env file contains a different `MONGODB_URI` to localhost, but whenever we run cypress tests we want to use the localhost database. That was my reason for explicitly defining it here rather than using the environment variables, but I'm not sure whether or not this was the right solution

Reply:
I'm not too familiar with how the deployments work and what set of environment variables these tests will use in CI, but hopefully they are set somewhere so that they can be configurable for these tests. I think it's probably worth looking into a bit more

Reply:
Discussed: it would be a pain right now to add the extra environment variables to both e2e-tests and github workflow, so we're not going to do it unless it becomes necessary :)